### ISSUE TITLE
rack update to 2.1.1 breaks warden

### ISSUE THREAD

**Issue by utkarsh2102** [wardencommunity/warden#187]
Hi,

When rack is updated to 2.1.1, it breaks warden; the test failures are here:

```
Failures:

  1) Warden::Manager thrown auth Failure should catch a resubmitted request
     Failure/Error: expect(result[2].body).to eq(["Bad"])
     
     NoMethodError:
       undefined method `body' for ["Bad"]:Array
     # ./spec/warden/manager_spec.rb:147:in `block (4 levels) in <top (required)>'

Finished in 0.25789 seconds (files took 0.23846 seconds to load)
224 examples, 1 failure

Failed examples:

rspec ./spec/warden/manager_spec.rb:105 # Warden::Manager thrown auth Failure should catch a resubmitted request
```

**Comment by ELepolt** [comment]
Also broken for 2.1.0, works for 2.0.9

**Comment by jsmestad** [comment]
This should be resolved with the latest `v1.2.9` release

**Comment by utkarsh2102** [comment]
Many thanks!
